Runtime error in latest version

I'm running into a runtime error after installing the latest release using brew.

`└─❱❱❱ tfm validate                                                                                                                                                      ─┘
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x28 pc=0x10455d884]

goroutine 1 [running]:
github.com/hashicorp/go-version.(*Version).String(0x0)
	/home/runner/go/pkg/mod/github.com/hashicorp/go-version@v1.6.0/version.go:369 +0x34
github.com/hashicorp/go-version.(*Version).Compare(0x1400024c140, 0x0)
	/home/runner/go/pkg/mod/github.com/hashicorp/go-version@v1.6.0/version.go:116 +0x34
github.com/hashicorp/go-version.(*Version).LessThan(...)
	/home/runner/go/pkg/mod/github.com/hashicorp/go-version@v1.6.0/version.go:308
github.com/rollwagen/tf-module-versions/internal/tf.Module.HasNewerVersion({{0x140000dcca0, 0x1d}, {{0x140000b6c50, 0xe}, 0x64}, {0x1400013cad2, 0x28}, {0x104796957, 0x3}, {0x1400013cad2, ...}})
	/home/runner/work/tf-module-versions/tf-module-versions/internal/tf/module.go:78 +0xc0
github.com/rollwagen/tf-module-versions/pkg/validater.Validate({0x104796648, 0x1}, {0x10479707f, 0x5}, 0x0)
	/home/runner/work/tf-module-versions/tf-module-versions/pkg/validater/main.go:104 +0x9c8
github.com/rollwagen/tf-module-versions/cmd.glob..func1(0x104c196c0?, {0x104796c29?, 0x0?, 0x0?})
	/home/runner/work/tf-module-versions/tf-module-versions/cmd/validate.go:17 +0x48
github.com/spf13/cobra.(*Command).execute(0x104c196c0, {0x104c4fe28, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x5b0
github.com/spf13/cobra.(*Command).ExecuteC(0x104c193e0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/rollwagen/tf-module-versions/cmd.Execute(...)
	/home/runner/work/tf-module-versions/tf-module-versions/cmd/root.go:20
main.main()
	/home/runner/work/tf-module-versions/tf-module-versions/main.go:6 +0x28
`

--- a/internal/tf/module.go
+++ b/internal/tf/module.go
@@ -66,7 +66,7 @@ func newVersion(versionAsString string) (*version.Version, error) {
 
 func (m Module) HasNewerVersion() bool {
 	ver := func(s string) *version.Version {
-		v, _ := version.NewVersion(s)
+		v, _ := newVersion(s)
 		return v
 	}
 

